expression/integration_test.go

```diff
@@ -637,6 +637,28 @@ func (s *testIntegrationSuite) TestTimeBuiltin(c *C) {
 	result.Check(testkit.Rows("0", "0", "0", "0", "0", "0", "0", "0"))
 }
 
+func (s *testIntegrationSuite) TestConvBuiltin(c *C) {
+	defer func() {
+		s.cleanEnv(c)
+		testleak.AfterTest(c)()
+	}()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	result := tk.MustQuery("SELECT CONV('a', 16, 2);")
```

Comment:
1. move this to TestMathBuiltin
2. use one line to test conv is ok,
result = tk.MustQuery("SELECT CONV('a', 16, 2), CONV('6E', 18, 8), ......")
result.Check(testkit.Rows("1010 172 ..."))